app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1572,98 +1564,154 @@ function DetailCells({ parent }) {
-            )}
-            {parent.render_column_header(
-              "log(1/E)",
-              `${parent.state.countType}_neglogevalue`,
-              `Average log-10-transformed expect value for alignments to NCBI NT/NR`
-            )}
-            {parent.render_column_header(
-              "%conc",
-              `${parent.state.countType}_percentconcordant`,
-              `Percentage of aligned reads belonging to a concordantly mappped pair (NCBI NT/NR)`,
-              parent.showConcordance
-            )}
-            <th>
-              <Tipsy content="Switch count type" placement="top">
-                <div className="sort-controls center left">
-                  <div
-                    className={
-                      parent.state.countType === "NT"
-                        ? "active column-switcher"
-                        : "column-switcher"
-                    }
-                    onClick={() => {
-                      parent.setState({ countType: "NT" });
-                    }}
-                  >
-                    NT
-                  </div>
-                  <div
-                    className={
-                      parent.state.countType === "NR"
-                        ? "active column-switcher"
-                        : "column-switcher"
-                    }
-                    onClick={() => {
-                      parent.setState({ countType: "NR" });
-                    }}
-                  >
-                    NR
+class ReportTableHeader extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      taxonModalData: null
+    };
+
+    this.openTaxonModal = this.openTaxonModal.bind(this);
+    this.closeTaxonModal = this.closeTaxonModal.bind(this);
+  }
+
+  renderTaxonModal() {
+    const { taxonModalData } = this.state;
+    if (!taxonModalData) return;
+
+    const { taxInfo, parent, taxonName } = taxonModalData;
+    return (
+      <TaxonModal
+        taxonId={taxInfo.tax_id}
+        taxonValues={{
+          NT: taxInfo.NT,
+          NR: taxInfo.NR
+        }}
+        parentTaxonId={
+          taxInfo.tax_level === 1 ? taxInfo.genus_taxid : undefined
+        }
+        background={{
+          name: parent.state.backgroundName,
+          id: parent.state.backgroundId
+        }}
+        taxonName={taxonName}
+        handleClose={this.closeTaxonModal}
+      />
+    );
+  }
+
+  openTaxonModal(taxonModalData) {
+    this.setState({
+      taxonModalData
+    });
+  }
+
+  closeTaxonModal() {
+    this.setState({
+      taxonModalData: null
+    });
+  }
+
+  render() {
+    const { parent } = this.props;
+
+    return (
+      <div className="reports-main">
+        {this.renderTaxonModal()}
+        <table id="report-table" className="bordered report-table">
+          <thead>
+            <tr className="report-header">
+              <th>
+                <span className={`table-arrow ""`}>
```

Comment:
I know this was here before but the "" doesn't do anything right? in `table-arrow ""`